examples/spark/src/main/scala/org/apache/carbondata/examples/CaseClassDataFrameAPIExample.scala

```diff
@@ -42,8 +45,25 @@ object CaseClassDataFrameAPIExample {
       .mode(SaveMode.Overwrite)
       .save()
 
+    val employee = List(Employee("anubhav", 20000))
```

Comment:
No I think it's alright I changed my code as u suggested and when I tested it ,it is working correctly, I am adding the test cases and will be pushing it